Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11-alpine
+
+WORKDIR /extract
+
+COPY . .
+
+RUN pip install -r requirements.txt
+
+RUN pip install qrcode[pil]
```

Comment:
This seems not necessary as it is already in requirements.txt
pil should be the same as Pillow.

Reply:
Fixed